docs/architecture/01_principles.md

```
@@ -150,10 +150,24 @@
 
 #### 1.6.1 적용 위치
 - **API Gateway**: 다운스트림 서비스 호출 시
-- **Payment Service**: PortOne API 호출 시
+- **User Service**: reCAPTCHA, PortOne 본인인증 API 호출 시
+- **Payment Service**: PortOne 결제 API 호출 시
 - **Reservation Service**: Feign 클라이언트로 Queue Service 호출 시
 
-#### 1.6.2 Resilience4j 설정 예시
+#### 1.6.2 예외 분류 및 서킷 상태 관리 정책
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)